bubblescan-client/src/App.tsx

```diff
@@ -18,7 +18,7 @@ function App() {
   // Function to send message to Flask
   const sendMessage = async () => {
     try {
-      const res = await fetch("http://localhost:5001/api/message", {
+      const res = await fetch("/api/message", {
```

Comment:
@rkarmuri These are the changes that might be breaking the dev version of the app. If that's the case, you'll need to parameterize the URL, and use http://localhost:5001 for dev and just blank for prod.

Reply:
Sure!!! You are right, I will create a dev branch and parameterize the URL to accept the localhost.